libmachine/drivers/utils.go

```diff
@@ -50,21 +50,19 @@ func RunSSHCommandFromDriver(d Driver, command string) (string, error) {
 	return output, nil
 }
 
-func sshAvailableFunc(d Driver) func() bool {
```

Comment:
The func returned by this func doesn't return any error, so it jus gets lost forever. This makes debugging SSH failures with Rancher Machine very difficult.